modules/setting/queue.go

```diff
@@ -142,6 +149,17 @@ func NewQueueService() {
 	if _, ok := sectionMap["LENGTH"]; !ok {
 		_, _ = section.NewKey("LENGTH", fmt.Sprintf("%d", Cfg.Section("mailer").Key("SEND_BUFFER_LEN").MustInt(100)))
 	}
+
+	// Handle the old test pull requests configuration
+	// Please note this will be a unique queue
+	section = Cfg.Section("queue.test_pull_requests")
```

Comment:
Maybe...
```suggestion
	section = Cfg.Section("queue.check_pull_requests")
```
"test" sounds like temporary code

Reply:
We call it a pull request patch test queue in config cheat sheet... I just named it after the current code. 

I guess it could be pr_patch_check? And refactor the function names to match?

Reply:
It's `check.go` and `models.PullRequestStatusChecking`, not `test.go` and `models.PullRequestStatusTesting` ... 🙄

Reply:
I've renamed it pr_patch_checker